src/sentry/utils/snuba.py

```diff
@@ -1025,6 +1025,18 @@ def _bulk_snuba_query(
                 raise SnubaError("Failed to parse snuba error response")
             raise UnexpectedResponseError(f"Could not decode JSON response: {response.data!r}")
 
+        if "quota_allowance" in body and body["quota_allowance"]:
+            with sentry_sdk.start_span(
+                op="snuba_query",
+                description=query_referrer,
+            ) as span:
+                quota_allowance_summary = body["quota_allowance"]["summary"]
+                span.set_tag("threads_used", quota_allowance_summary["threads_used"])
+                for k, v in quota_allowance_summary["throttled_by"].items():
+                    span.set_tag(k, v)
+                for k, v in quota_allowance_summary["rejected_by"].items():
+                    span.set_tag(k, v)
+
         if response.status != 200:
```

Comment:
are you going to add the details to the error case in this PR or a separate one?

Reply:
what do you mean by error case? if you're referring to adding the details when `response.status == 429`, then `error["message"]` already contains the summary. On Sentry issues, the summary is currently displayed

<img width="908" alt="Screenshot 2024-06-26 at 3 26 23 PM" src="https://github.com/getsentry/sentry/assets/159840875/3d13885b-2887-4273-a51a-72cf28e721ac">

Reply:
some of these things should be tags. e.g.

threads_used, rejected_by_policy, rejected_by_quota_used, rejected_by_quota_unit

that makes them searchable in the sentry product